src/index.js

```diff
@@ -17,7 +19,8 @@ function init( {
 			createPopup( popupContainer ),
 		events = customEvents( popup ),
 		last = {},
-		showPopup = ( e, refresh = false ) => {
+		showPopup = ( e, refresh = false, popupId = Date.now() ) => {
```

Comment:
Is `showPopup()` ever called with a `popupId`?

Reply:
nope, i'm thinking of where's the best place to set it.

Reply:
Can't you do `currentPopupId = Date.now()` directly?

Reply:
Done